operations/app/src/environments/tf

```diff
@@ -165,6 +183,18 @@ if [[ "${ACTUAL_RESULTS?}" == "${EXPECTED_RESULTS?}" ]]; then
         ACTUAL_RESULTS="${ACTUAL_RESULTS?}${?}"
         ;;
 
+    plan-file*)
+        # Custom command we made for the purpose of CI/CD
+        EXPECTED_RESULTS="${EXPECTED_RESULTS?}0"
+        terraform plan \
+            --var-file "${TFVARS?}" \
+            >/dev/null -out=${PLAN_DIR?}/${ENVIRONMENT?}-${HERE?}.tfplan
+        terraform show -no-color \
+            ${PLAN_DIR?}/${ENVIRONMENT?}-${HERE?}.tfplan \
+            > ${PLAN_DIR?}/${ENVIRONMENT?}-${HERE?}.txt
+        ACTUAL_RESULTS="${ACTUAL_RESULTS?}${?}"
+        ;;
```

Comment:
This is the block of code I am most concerned with from a security perspective. I want to ensure that under all scenarios, Terraform will never log sensitive information to the console.

Reply:
I do know that terraform will block 'sensitive' values but I don't know of the top of my head if there are edge cases where they might leak.
I'm with you

Reply:
Also capture the error as well.

Reply:
Fixed in next push